### PR TITLE
libxml++: add license

### PR DESCRIPTION
**Pull request by janesmae**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libxml++.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Libxmlxx < Formula
   homepage "https://libxmlplusplus.sourceforge.io/"
   url "https://download.gnome.org/sources/libxml++/2.42/libxml++-2.42.0.tar.xz"
   sha256 "3d032aede98a033eb5e815b4bfa9fa7b4e745268e6fd1ce8b1d0f70bcaf4736d"
+  license "LGPL-2.0-or-later"
```
Comment (miccal) [inline Formula/libxml++.rb:6]: I think this should be `LGPL-2.1-only`, according to https://gitlab.gnome.org/GNOME/libxmlplusplus/-/blob/master/COPYING

Comment (janesmae) [reply Formula/libxml++.rb:6]: And their website states: Libxml++ is released under the LGPL version 2 or above

Ref: http://libxmlplusplus.sourceforge.net

Comment (miccal) [reply Formula/libxml++.rb:6]: @chenrui333 thoughts on this one?

Comment (SMillerDev) [reply Formula/libxml++.rb:6]: I'd say we should ask upstream. It seems like they're not sure either.

Comment (chenrui333) [reply Formula/libxml++.rb:6]: yeah, let's ask in the upstream. We can also even tag the maintainer if they are also on github. :)

**Pull request closed without merging**